lib/digest/crc5.rb

```diff
@@ -12,37 +12,26 @@ class CRC5 < CRC
 
     XOR_MASK = 0x1f
 
-    CRC_MASK = (0x1f << 3)
```

Comment:
According to [pycrc-0.10.0](https://pycrc.org/)'s (reference implementation) generated C code, the CRC "mask" should be `0x1f`.

```shell
/src/pycrc.py --generate c -o crc5.c --model crc-5 --algorithm=table-driven
```

```c
crc_t crc_update(crc_t crc, const void *data, size_t data_len)
{
    const unsigned char *d = (const unsigned char *)data;
    unsigned int tbl_idx;

    while (data_len--) {
        tbl_idx = crc ^ *d;
        crc = (crc_table[tbl_idx] ^ (crc >> 8)) & 0x1f;
        d++;
    }
    return crc & 0x1f;
}
```

Reply:
Hmmm, I understand. I will try to keep it in this PR.
However, it will eventually become unnecessary in later PRs.